VideoAPI/VideoApi/VideoApi.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
```

Comment:
Could you explain why the solution file has changed please?

Reply:
VS doing something, it was created the second one in the project directory. It's deleted now.